frame/treasury/src/lib.rs

```diff
@@ -393,6 +395,33 @@ pub mod pallet {
 				.map_err(|_| Error::<T, I>::TooManyApprovals)?;
 			Ok(())
 		}
+
+		/// Force a previously approved proposal to be removed from the approval queue.
+		///
+		/// May only be called from `T::RejectOrigin`.
+		///
```

Comment:
We should document the possible errors that this dispatchable can return. In this case, it's only one, but it'll still need a description of what the error means.

Reply:
Hi @KiChjang, there's a short description of what the error means under the error enum:

```
/// Proposal has not been approved.
		ProposalNotApproved,
```

Is there an extra documentation requirement left out please?

Reply:
Yes, you'll need docs for this dispatchable function. Think about what it looks like from a Rust developer's perspective -- they're curious about this function and they mouse over one of the call sites for this function. Their IDE then displays a tooltip about what this function is about, and it's exactly this doc comment, which does not inform them about the possible errors that it can return. Not everyone is going to click through to see the function definition to know that there could possibly be an error that gets returned, and even less would be bothered to look up the pallet error definition to find the docs for `ProposalNotApproved`.

Reply:
What Keith is asking for is naming the errors that could be returned in this doc, not explaining what the error _is_.

Reply:
See https://github.com/paritytech/cumulus/blob/d5434b3054b14bdb77e63d803474e400b932f52b/pallets/dmp-queue/src/lib.rs#L131-L142 for an example of how to write well-formatted and informative rust docs for pallet dispatchables.

Reply:
Thanks @KiChjang and @kianenigma, I've updated this.